imageio/__main__.py

```diff
@@ -0,0 +1,137 @@
+
+
+def download_bin_main():
+    """ Argument-parsing wrapper for `download_bin` """
+    description = "Download plugin binary dependencies"
+    phelp = "Plugin name for which to download the binary. "\
+            + "If no argument is given, all binaries are downloaded."
+    dhelp = "Download the binaries to the package directory "\
+            + "(default is the users application data directory). "\
+            + "This might require administrative rights."
+    example_text = "examples:\n"\
+                   + "  imageio_download_bin all\n"\
+                   + "  imageio_download_bin ffmpeg\n"\
+                   + "  imageio_download_bin avbin ffmpeg\n"
+    parser = argparse.ArgumentParser(
+        description=description,
+        epilog=example_text,
+        formatter_class=argparse.RawDescriptionHelpFormatter)
+    parser.add_argument("plugin", type=str, nargs="*", default="all",
+                        help=phelp)
+    parser.add_argument("--package-dir", dest="package_dir",
+                        action="store_true", default=False, help=dhelp)
+    args = parser.parse_args()
+    download_bin(plugin_names=args.plugin, package_dir=args.package_dir)
+
+
+def remove_bin(plugin_names=["all"]):
+    """ Remove binary dependencies of plugins
+    
+    This is a convenience method that removes all binaries
+    dependencies for plugins downloaded by imageio.
+    
+    Notes
+    -----
+    It only makes sense to use this method if the binaries
+    are corrupt.
+    """
+    if plugin_names.count("all"):
+        # Use all plugins
+        plugin_names = PLUGINS_WITH_BINARIES
+
+    print("Removing binaries for: {}.".format(", ".join(plugin_names)))
+
+    rdirs = util.resource_dirs()
+
+    for plg in plugin_names:
+        msg = "Plugin {} is not registered for binary download!".format(plg)
+        assert plg in PLUGINS_WITH_BINARIES, msg
+    
+    for rd in rdirs:
+        # plugin name is in subdirectories
+        for rsub in os.listdir(rd):
+            if rsub in plugin_names:
+                shutil.rmtree(op.join(rd, rsub))
+
+
+def remove_bin_main():
+    """ Argument-parsing wrapper for `remove_bin` """
+    description = "Remove plugin binary dependencies"
+    phelp = "Plugin name for which to remove the binary. "\
+            + "If no argument is given, all binaries are removed."
+    example_text = "examples:\n"\
+                   + "  imageio_remove_bin all\n"\
+                   + "  imageio_remove_bin ffmpeg\n"\
+                   + "  imageio_remove_bin avbin ffmpeg\n"
+    parser = argparse.ArgumentParser(
+        description=description,
+        epilog=example_text,
+        formatter_class=argparse.RawDescriptionHelpFormatter)
+    parser.add_argument("plugin", type=str, nargs="*", default="all",
+                        help=phelp)
+    args = parser.parse_args()
+    remove_bin(plugin_names=args.plugin)
+
+
+if __name__ == "__main__":
+    download_bin_main()
```

Comment:
For testing and dev, it might be nice to support `python -m imageio download_bin` and `python -m imageio remove_bin`. As it is now, I think one could call `python -m imageio` and it would default to `download_bin_main()`. In short something along these lines should work:
```py
if __name__ == "__main__":  # technically always True for a __main__.py
    if len(sys.argv) > 1 and sys.argv[1] == 'download_bin':
        download_bin_main()
    elif len(sys.argv) > 1 and sys.argv[1] == 'remove_bin':
        remove_bin_main()
    else:
        raise RuntimeError('Invalid use of the imageio CLI')
```